packages/hoprd/crates/hoprd-hoprd/Cargo.toml

```diff
@@ -63,4 +64,12 @@ wasm = [
     "utils-metrics?/wasm",
     "utils-types/wasm",
 ]
-prometheus = [ "dep:utils-metrics", "hopr-lib/prometheus", "core-transport/prometheus", "core-strategy/prometheus", "core-network/prometheus", "core-ethereum-indexer/prometheus" ]
+prometheus = [
+    "dep:utils-metrics",
+    "dep:lazy_static",
+    "hopr-lib/prometheus",
+    "core-transport/prometheus",
+    "core-strategy/prometheus",
+    "core-network/prometheus",
+    "core-ethereum-indexer/prometheus"
```

Comment:
These should be later reworked so that the types and functionality is reexported by hopr-lib.